logs-lib/logs/panels.libsonnet

```diff
@@ -23,11 +23,10 @@ function(
       + timeSeries.queryOptions.withDatasource(
         logsVolumeTarget.datasource.type, logsVolumeTarget.datasource.uid
       )
+      + timeSeries.queriesSelector.withMaxDataPoints(100)
       + custom.withDrawStyle('bars')
       + custom.stacking.withMode('normal')
       + custom.withFillOpacity(50)
-      // should be set, otherwise interval is around 1s by default
-      + timeSeries.queryOptions.withInterval('30s')
```

Comment:
Is this no longer required?

Reply:
yes, interval is calculated based on maxdatapoints now, which works better.